execution/src/types.rs

```diff
@@ -1,3 +1,30 @@
+use anyhow::{bail, Result};
+
 pub type Address = String;
-pub type Bytecode = Vec::<u8>;
-pub type Ledger = std::collections::BTreeMap<Address, Bytecode>; // Byttecode instead of String
\ No newline at end of file
+pub type Bytecode = Vec<u8>;
+pub type Ledger = std::collections::BTreeMap<Address, Bytecode>; // Byttecode instead of String
+
+/// That's what is returned when a module is executed correctly since the end
+pub struct Response {
+    /// returned value from the module call
+    pub ret: String,
+    /// number of points that remain after the execution (metering)
+    pub remaining_points: u64,
+}
+
+#[derive(Clone)]
+pub struct Interface {
```

Comment:
The interface does not provide calls like get_data, write_data, random, time etc will they be there or somewhere else ?

Reply:
> "All the interface is not implemented and such functions as "get_time" have to be done in another PR." - @adrien-zinger 

Ok I guess that we can do this in this PR. I'll add:
- get_data
- write_data
- get_time
- random